python/publish-layers.sh

```diff
@@ -34,7 +34,7 @@ function build-python27 {
     echo "Building New Relic layer for python2.7"
     rm -rf $BUILD_DIR $PY27_DIST
     mkdir -p $DIST_DIR
-    pip install --no-cache-dir -qU newrelic-lambda -t $BUILD_DIR/lib/python2.7/site-packages
+    pip install --no-cache-dir -qU newrelic newrelic-lambda -t $BUILD_DIR/lib/python2.7/site-packages
```

Comment:
Is the change from `newrelic-lambda` to `newrelic newrelic-lambda` a result of a change in namespacing?

Reply:
No, I'm just explicitly including the `newrelic` package here with the `-U` flag to tell pip to update both to latest versions.